regression/scripts/VTOL/test_VTOL.py

```diff
@@ -0,0 +1,109 @@
+# test_VTOL.py
+# 
+# Created:  July 2018, M. Clarke
+#
+""" setup file for a mission with a QuadShot
```

Comment:
the regression progression here is a bit confusing?  does slip stream call test_vtol?  if not, this isnt called.